src/phoenix/db/migrations/versions/10460e46d750_datasets.py

```diff
@@ -198,7 +198,7 @@ def upgrade() -> None:
             sa.String,
             nullable=True,
         ),
-        sa.Column("output", JSON_, nullable=True),
+        sa.Column("output", JSON_, nullable=False),
```

Comment:
SO we will have { result: null } for error'd runs here?

Reply:
It's a bit more nuanced because both is technically possible: `{}` or `{ result: null }`, but yes, the latter is what will be entered into the database, but we couldn't eliminate the former entirely as a possibility. But in gql we would just do `output.get("result")` so it all works out either way.

Reply:
note that it'll always be a dict no matter what, so we eliminated the type error at least

Reply:
`None` is automatically converted to `{}` by the sqlalchemy because we told it to